core/init/router/router.go

```diff
@@ -4,8 +4,8 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/1Panel-dev/1Panel/cmd/server/docs"
-	"github.com/1Panel-dev/1Panel/cmd/server/web"
+	"github.com/1Panel-dev/1Panel/core/cmd/server/docs"
+	"github.com/1Panel-dev/1Panel/core/cmd/server/web"
 	"github.com/1Panel-dev/1Panel/core/global"
 	"github.com/1Panel-dev/1Panel/core/i18n"
 	"github.com/1Panel-dev/1Panel/core/middleware"
```

Comment:
In terms of checking code quality and identifying potential inconsistencies in the provided snippet, there appears to be no known irregularities or specific optimization suggestions related to the context mentioned (knowledge cutoff: 2021-09-01). This information does not apply to the current date (2024-12-17), so I will answer directly with respect to the given context.

As an AI designed by OpenAI, my capabilities are limited to understanding previous versions and environments. For up-to-date programming guidelines and best practices, it's always recommended to consult current documentation on languages you're working with, like Go or Python. However, based solely on the examples you've shared until 2021, I do not see any apparent issues that would require immediate attention.